gdbserver_tests test does not work at all
simply waits for eternity

--- valgrind-3.22.0/Makefile.in	2023-12-26 18:16:46.161022244 +0100
+++ valgrind-3.22.0/Makefile.in.new	2023-12-26 19:05:09.563683141 +0100
@@ -1435,18 +1435,18 @@
 
 regtest: check
 	$(top_srcdir)/gdbserver_tests/make_local_links $(GDB)
-	if @PERL@ tests/vg_regtest gdbserver_tests $(TOOLS) $(EXP_TOOLS) ; then \
-	   tests/post_regtest_checks $(abs_top_srcdir) gdbserver_tests $(TOOLS) $(EXP_TOOLS); \
+	if @PERL@ tests/vg_regtest $(TOOLS) $(EXP_TOOLS) ; then \
+	   tests/post_regtest_checks $(abs_top_srcdir) $(TOOLS) $(EXP_TOOLS); \
 	else \
 	   tests/post_regtest_checks $(abs_top_srcdir) gdbserver_tests $(TOOLS) $(EXP_TOOLS); \
 	   false; \
 	fi
 post-regtest-checks:
-	tests/post_regtest_checks $(abs_top_srcdir) gdbserver_tests $(TOOLS) $(EXP_TOOLS)
+	tests/post_regtest_checks $(abs_top_srcdir) $(TOOLS) $(EXP_TOOLS)
 nonexp-regtest: check
 	@PERL@ tests/vg_regtest $(TOOLS)
 exp-regtest: check
-	@PERL@ tests/vg_regtest gdbserver_tests $(EXP_TOOLS)
+	@PERL@ tests/vg_regtest $(EXP_TOOLS)
 # Nb: gdbserver_tests are put in exp-regtest rather than nonexp-regtest
 # because they are tested with various valgrind tools, so might be using
 # an experimental tool.
--- valgrind-3.22.0/Makefile.am	2023-10-31 18:25:52.000000000 +0100
+++ valgrind-3.22.0/Makefile.am.new	2023-12-26 19:06:33.986624317 +0100
@@ -76,18 +76,18 @@
 ## Ensure make exits with error if PERL fails or post_regtest_checks fails.
 regtest: check
 	$(top_srcdir)/gdbserver_tests/make_local_links $(GDB)
-	if @PERL@ tests/vg_regtest gdbserver_tests $(TOOLS) $(EXP_TOOLS) ; then \
-	   tests/post_regtest_checks $(abs_top_srcdir) gdbserver_tests $(TOOLS) $(EXP_TOOLS); \
+	if @PERL@ tests/vg_regtest $(TOOLS) $(EXP_TOOLS) ; then \
+	   tests/post_regtest_checks $(abs_top_srcdir) $(TOOLS) $(EXP_TOOLS); \
 	else \
 	   tests/post_regtest_checks $(abs_top_srcdir) gdbserver_tests $(TOOLS) $(EXP_TOOLS); \
 	   false; \
 	fi
 post-regtest-checks:
-	tests/post_regtest_checks $(abs_top_srcdir) gdbserver_tests $(TOOLS) $(EXP_TOOLS)
+	tests/post_regtest_checks $(abs_top_srcdir) $(TOOLS) $(EXP_TOOLS)
 nonexp-regtest: check
 	@PERL@ tests/vg_regtest $(TOOLS)
 exp-regtest: check
-	@PERL@ tests/vg_regtest gdbserver_tests $(EXP_TOOLS)
+	@PERL@ tests/vg_regtest $(EXP_TOOLS)
 # Nb: gdbserver_tests are put in exp-regtest rather than nonexp-regtest
 # because they are tested with various valgrind tools, so might be using
 # an experimental tool.
